source/includes/steps-starting-vsce-connect-with-advanced-connection-settings.yaml

```diff
@@ -157,17 +157,17 @@ content: |
                specify the :guilabel:`Hostname`. You do not need to 
                specify a port.
 
-               .. note::
+              
 
-                  You can only use SRV connection strings to connect to 
-                  deployments running MongoDB 3.6 and later.
+               You can only use SRV connection strings to connect to 
```

Comment:
[suggestion]

I think we can consider removing this line entirely since this covers all supported versions.

Reply:
@jmd-mongo this look good?